arch/x86/kernel/cpu/resctrl/monitor.c

```diff
@@ -416,9 +434,11 @@ static void add_rmid_to_limbo(struct rmid_entry *entry)
 	}
 	put_cpu();
 
-	if (entry->busy)
+	if (entry->busy) {
 		rmid_limbo_count++;
-	else
+		if (IS_ENABLED(CONFIG_RESCTRL_RMID_DEPENDS_ON_CLOSID))
+			closid_num_dirty_rmid[entry->closid]++;
+	} else
```

Comment:
at upstream they have the open and close brace after the else. Reference 
https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/?id=b30a55df60c35df09b9ef08dfb0a0cbb543abe81

Reply:
@clsotog I have cleaned up.
the base kernel is [0].

[0], https://github.com/NVIDIA-BaseOS-6/linux-nvidia-6.5/tree/MPAM-dev